torch/testing/_internal/common_methods_invocations.py

```diff
@@ -651,6 +651,21 @@ def sample_inputs_amax_amin(op_info, device, dtype, requires_grad):
                              args=args)
                  for size, args in test_cases)
 
+def sample_inputs_argmax_argmin(op_info, device, dtype, requires_grad):
```

Comment:
@imaginary-person you need an `OpInfo` entry that's run all the tests for these sample inputs

Reply:
Oops! I missed it while manually merging changes. Thanks for catching it!